### PR TITLE
Scylla tracing negotiation

### PR DESCRIPTION
**Pull request by Deexie**

### CODE REVIEW

File: driver-core/src/main/java/com/datastax/driver/core/Connection.java [hunk 1]
```diff
@@ -1288,7 +1295,7 @@ Connection open(Host host)
       Connection connection = new Connection(buildConnectionName(host), endPoint, this);
       // This method opens the connection synchronously, so wait until it's initialized
       try {
-        connection.initAsync().get();
+        connection.initAsync(false).get();
```
Comment (Deexie) [inline driver-core/src/main/java/com/datastax/driver/core/Connection.java:1298]: Take a notice of this line, please

File: driver-core/src/main/java/com/datastax/driver/core/HostConnectionPool.java [hunk 2]
```diff
@@ -301,6 +306,7 @@ public void run() {
     List<Connection> newConnections = manager.connectionFactory().newConnections(this, toCreate);
     connections.addAll(newConnections);
 
+    boolean tracingRequested = !(manager.getTracingInfoFactory() instanceof NoopTracingInfoFactory);
```
Comment (wprzytula) [inline driver-core/src/main/java/com/datastax/driver/core/HostConnectionPool.java]: Why don't you use here the method you wrote above?

**Review by wprzytula — COMMENTED** [review]
WET

File: driver-core/src/main/java/com/datastax/driver/core/HostConnectionPool.java [hunk 3]
```diff
@@ -301,6 +306,7 @@ public void run() {
     List<Connection> newConnections = manager.connectionFactory().newConnections(this, toCreate);
     connections.addAll(newConnections);
 
+    boolean tracingRequested = isTracingRequested();
```
Comment (wprzytula) [inline driver-core/src/main/java/com/datastax/driver/core/HostConnectionPool.java]: Either at least add `final` here, or even get rid of the variable here entirely - you can call the function directly where you need this value below.

**Review by wprzytula — COMMENTED** [review]
Small style hint

**Review by margdoc — APPROVED** [review]
LGTM

**Review by wprzytula — APPROVED** [review]
LGTM

**Pull request closed without merging**